deps.toml

```diff
@@ -147,6 +153,7 @@ micronaut-validation = { module = "io.micronaut:micronaut-validation" }
 
 [bundles]
 apache = ["apache-commons", "apache-commons-lang"]
+aws = ["aws-s3","aws-sts","aws-auth","aws-iam"]
```

Comment:
Same as above - these deps have been included in build.gradle for airbytehq/airbyte and airbytehq/airbyte-platform. Do we need them in both places?